src/routes/Search.tsx

```diff
@@ -281,7 +281,15 @@ function ActualSearch() {
                         {(contact) => (
                             <button
                                 onClick={() => sendToContact(contact)}
-                                class="flex items-center gap-2"
+                                class={
```

Comment:
if you're using logic for the style you want to use `classList`

put the default style in class, and then add

```
classList={{ "text-white/20 opacity-60": !contact.ln_address && !contact.lnurl }}
```